stdlib/int32.gr

```diff
@@ -138,7 +136,11 @@ export let rec mod = (x: Int32, y: Int32) => {
     let yabs = abs(yval)
     let mval = WasmI32.remS(xabs, yabs)
     let mres = WasmI32.sub(yabs, mval)
-    newInt32(if (WasmI32.ne(mval, 0n)) (if (WasmI32.ltS(yval, 0n)) WasmI32.sub(0n, mres) else mres) else 0n)
+    newInt32(
+      if (WasmI32.ne(mval, 0n)) (
+        if (WasmI32.ltS(yval, 0n)) WasmI32.sub(0n, mres) else mres
+      ) else 0n
```

Comment:
I'm good this with. Highlighting for @ospencer to sign-off.

Reply:
This looks good!